File: 1.TwoSum/in Python/Brutforce.py
Problem : Given an array of integers nums and an integer target, return indices of the two numbers such that they add up to target.

Solution: iterating  through each combination  to find the required solve.

Code:
    def twoSum(self, nums, target):
        for i in range(len(nums)):
            for j in range(i + 1, len(nums)): 
                if nums[i] + nums[j] == target:
                    return [i, j] 
        return []  
 

File: 1.TwoSum/In Dart/BrutForce.dart
Problem : Given an array of integers nums and an integer target, return indices of the two numbers such that they add up to target.
Solution:
Code:
class Solution {
  List<int> twoSum(List<int> nums, int target) {
 
    for (int i = 0; i < nums.length; i++) {
      for (int j = i + 1; j < nums.length; j++) {
      
        if (nums[i] + nums[j] == target) {
          return [i, j];
        }
      }
    }
  
    return [];
  }
}
